### PR TITLE
rebase: update go-ceph to v0.5.0

### PR DESCRIPTION
**Pull request by Madhu-1**
as go-ceph is 0.5.0 is released updating the dependency to the latest release. more info about the release at https://github.com/ceph/go-ceph/releases/tag/v0.5.0

Updates: #1260

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -64,6 +64,8 @@ github.com/cenkalti/backoff v2.1.1+incompatible/go.mod h1:90ReRw6GdpyfrHakVjL/QH
 github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
 github.com/ceph/go-ceph v0.4.0 h1:KJsT6j1IbsEtui3ZtDcZO//uZ+IVBNT6KO7u9PuMovE=
 github.com/ceph/go-ceph v0.4.0/go.mod h1:wd+keAOqrcsN//20VQnHBGtnBnY0KHl0PA024Ng8HfQ=
+github.com/ceph/go-ceph v0.5.0 h1:x5VmFq19Op6DjzWuxAUG3wZZoC3L160Rt6pJOOiRfW0=
+github.com/ceph/go-ceph v0.5.0/go.mod h1:wd+keAOqrcsN//20VQnHBGtnBnY0KHl0PA024Ng8HfQ=
```
Comment (nixpanic) [inline go.sum:68]: Interestingly #1361 deletes the lines for go-ceph v0.4.0!? The rest of the PRs look exactly the same.

Comment (Madhu-1) [reply go.sum:68]: May be due to the golang version difference? Let me recheck

Comment (Madhu-1) [reply go.sum:66]: this is fixed now (i missed running `go mod tidy`)

**Comment by nixpanic** [comment]
> /skip-doc-change

This is testing a CI job **ignore/testing/skip-doc-change** which should start some testing, and not return early.

**Comment by Madhu-1** [comment]
@mergifyio rebase

**Comment by Madhu-1** [comment]
>Unable to rebase: user ceph-csi-bot is unknown. Please make sure ceph-csi-bot has logged in Mergify dashboard.
err-code: DFF2B

@nixpanic we need to login to Mergify Dashboard with  ceph-csi-bot

**Comment by Madhu-1** [comment]
@Mergifyio rebase

**Comment by Madhu-1** [comment]
@humblec PTAL

**Pull request merged**